scripts/st2bootstrap-deb.sh

```diff
@@ -329,6 +329,10 @@ install_st2() {
   sudo crudini --set /etc/st2/st2.conf database password "${ST2_MONGODB_PASSWORD}"
 
   sudo st2ctl start
+  # TODO: Fix https://github.com/StackStorm/st2-packages/issues/445 (under xenial register content fails on first boot)
```

Comment:
Could debug that thing deeper here: https://github.com/StackStorm/st2-packages/issues/445#issuecomment-286778253 with bash script to reproduce it every time.

It's something how st2 services interact with mongo. This happens on a clean MongoDB during the first st2 register rules run.

@Kami I hope to get your help there, since after further digging it looks not like a Mongo startup or packaging/services problem, but more like an issue to fix in StackStorm core.